homeassistant/components/devolo_home_network/sensor.py

```diff
@@ -32,10 +31,9 @@
     PLC_RX_RATE,
     PLC_TX_RATE,
 )
+from .coordinator import DevoloDataUpdateCoordinator
 from .entity import DevoloCoordinatorEntity
 
-PARALLEL_UPDATES = 1
```

Comment:
`PARALLEL_UPDATES` should be set to `0` if that's not needed anymore

Reply:
But it didn't have that constant before https://github.com/home-assistant/core/pull/122847 yet being platinum. And it's 0 the default value? I don't mind readding the constant, if needed, I just wanted to keep it as simple as possible.

Reply:
0 is the default, but then we at least have it set explicitly

Reply:
ok, I added the constant again.